packages/example-app-base/src/components/ApiReferencesTable/ApiReferencesTable.tsx

```diff
@@ -0,0 +1,497 @@
+
+    return this._parseILinkTokens(true, extendsTokens);
+  }
+
+  private _renderDescription(): JSX.Element | undefined {
+    const { description } = this.props;
+
+    return description && description !== '' ? (
+      <Text variant={'medium'}>
+        <div dangerouslySetInnerHTML={{ __html: description }} />
+      </Text>
+    ) : (
+      undefined
+    );
+  }
+
+  private _renderTitle(): JSX.Element | undefined {
+    const { title, name } = this.props;
+
+    return title ? (
+      <Text variant={'xLarge'}>
+        <div dangerouslySetInnerHTML={{ __html: title }} id={name} />
+      </Text>
+    ) : (
+      undefined
+    );
+  }
+
+  private _onRenderRow(props: IDetailsRowProps, defaultRender?: IRenderFunction<IDetailsRowProps>): JSX.Element {
+    const { item } = props;
+    const rowStyles: Partial<IDetailsRowStyles> = {
+      root: {
+        selectors: {
+          ':hover': {
+            background: 'transparent'
+          }
+        }
+      },
+      isMultiline: {
+        wordBreak: 'break-word'
+      }
+    };
+
+    const deprecatedRowStyles = mergeStyles([
+      {
+        background: '#ffffcc'
+      },
+      'DetailsRow-deprecated'
+    ]);
+
+    if (item.deprecated === true) {
+      const deprecatedStyles: Partial<IDetailsRowStyles> = {
+        root: {
+          background: '#FFFFCC',
+          selectors: {
+            ':hover': {
+              background: '#FFFFCC'
+            }
+          }
+        }
+      };
+
+      return (
+        <div className={deprecatedRowStyles}>
+          Warning. This API is now obsolete.
+          {defaultRender!({
+            ...props,
+            styles: rowStyles && deprecatedStyles
+          })}
+        </div>
+      );
+    }
+
+    return <DetailsRow {...props} styles={rowStyles} />;
+  }
+
+  private _parseILinkTokens(extend: boolean, linkTokens?: ILinkToken[]): JSX.Element | undefined {
+    if (linkTokens && linkTokens.length > 0) {
+      if (extend) {
+        return (
+          <Text variant={'small'} className="ApiReferencesTable-extends">
+            {'Extends '}
+            {linkTokens.map((token: ILinkToken, index: number) => {
+              let match: RegExpMatchArray | null = null;
+              let hash: string = '/components/';
+              if (this._baseUrl !== '') {
+                match = this._baseUrl.match(/\/\w+\//);
+                if (match !== null) {
+                  hash = match[0];
+                }
+              }
+              if (token.pageKind && token.hyperlinkedPage) {
+                let href;
+                if (token.pageKind === 'References') {
+                  href = '#' + hash + token.pageKind.toLowerCase() + '/' + token.hyperlinkedPage.toLowerCase() + '#' + token.text;
+                } else {
+                  href = '#' + hash + token.hyperlinkedPage.toLowerCase() + '#' + token.text;
+                }
+                return (
+                  <Link href={href} key={token.text + index}>
```

Comment:
Please don't use the index as a key. If you can combine the text and some other property of the token to use as the key, that would be better. (Same comment below.)